package/MDAnalysis/transformations/rotate.py

```diff
@@ -69,11 +70,14 @@ def rotateby(angle, direction, point=None, center="geometry", wrap=False, ag=Non
         vector that will define the direction of a custom axis of rotation from the
         provided point.
     ag: AtomGroup, optional
-        use this to define the center of mass or geometry as the point from where the
-        rotation axis will be defined
-    center: str, optional
-        used to choose the method of centering on the given atom group. Can be 'geometry'
-        or 'mass'
+        use the eighted center of an AtomGroup as the point from where the rotation axis
+        will be defined
+    weights: {"mass", ``None``} or array_like, optional
+        define the weights of the atoms when calculating the center of the AtomGroup.
+        With ``"mass"`` uses masses as weights; with ``None`` weigh each atom equally.
+        If a float array of the same length as `ag` is provided, use each element of
+        the `array_like` as a weight for the corresponding atom in `ag`. Default is 
+        None.
     wrap: bool, optional
```

Comment:
shouldn't this follow the atomgroup convention to call it `pbc`?

Reply:
@richardjgowers proposed the change to `wrap` in #1946. The argument is explained in https://github.com/MDAnalysis/mdanalysis/issues/1185#issuecomment-360478631 and I agree with it. Since there's no code built upon these transformations yet, there's no risk of breaking things.

Reply:
yes, there is a risk of breaking user expectation.  As a newbie using MDAnalysis I would be confusing why the transformation has a `wrap` argument and the corresponding atomgroup method a `pbc` argument. This is not an intuitive design. 

Besides the comment also mentions to deprecate the pbc keyword argument. Is anyone on that yet?

Reply:
@davidercruz did you start work on introducing the wrap keyword in the other atomgroup functions?

Reply:
@kain88-de not yet

Reply:
Yeah sorry, I didn't want to propagate the `pbc` keyword any more.  I've started looking at the `unwrap` keyword. Adding the `wrap` keyword should just be deprecating `pbc` and aliasing it to `wrap`